src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListUpdater.java

```diff
@@ -71,8 +75,12 @@ public void processDailyList(DailyListEntity dailyListEntity) throws JsonProcess
                             courtHouseName, sitting.getCourtRoomNumber(),
                             caseNumber, dailyListHearing.getHearingDetails().getHearingDate()
                         );
+                        hearing.setCreatedBy(systemUserHelper.getSystemUser());
```

Comment:
`DailyList Processor` user account is -2, will this bring back a different one?

Reply:
ye you are right it should be a dailylist processor user, with the userId 1. Accept.